examples/connectivity/plot_power_connectome.py

```diff
@@ -0,0 +1,73 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from nilearn import datasets
+
+
+#fetch power atlas coords
+power = datasets.fetch_atlas_power_2011()
+power_coords = power.rois[['x', 'y', 'z']]
+
+#fetch dataset
+
+adhd = datasets.fetch_adhd(n_subjects=1)
+
+# Preprocess image and extract signals from Power-264
+from nilearn import input_data
+masker = input_data.NiftiSpheresMasker(seeds = power_coords,
+                                       radius = 5.,
+                                       standardize = True,
+                                       smoothing_fwhm = 4,
+                                       detrend = True,
+                                       low_pass = 0.1,
+                                       high_pass = 0.01,
+                                       t_r = 2.5,
+                                      verbose = 5)
```

Comment:
Could you install a pep8 checker: this code does not satisfy pep8 standard (it has whitespace surrounding equal signs in the keyword arguments).